api/src/main/java/org/pmiops/workbench/api/ConceptsController.java

```diff
@@ -20,6 +23,26 @@
 @RestController
 public class ConceptsController implements ConceptsApiDelegate {
 
+  private static final ImmutableMultimap<Domain, String> DOMAIN_MAP =
```

Comment:
We may want to in future have this kind of mapping in our CDR schema somewhere... I'm not sure. For now, hardcoding it here seems OK-ish.

My assumption is that researchers are going to want to filter concepts by the enum values shown here, and not have to choose from underlying domain IDs (which in a number of cases could be for two different domains.)